src/lib/d.ts

```diff
@@ -0,0 +1,72 @@
+interface GenderPercents {
+  male: number;
+  female: number;
+  other: number;
+  not_say: number;
+}
+
+interface RacePercents {
+  asian: number;
+  black_or_african_american: number;
+  native_american_or_alaskan_native: number;
+  native_hawaiian_or_other_pacific_islander: number;
+  white: number;
+  other: number;
+  not_say: number;
+}
+
+interface LatinxPercents {
+  yes: number;
+  no: number;
+  not_say: number;
+}
+
+interface ReferenceData {
+  gender: GenderPercents;
+  race: RacePercents;
+  is_hispanic_latinx: LatinxPercents;
+}
+
+type GenderResponse = "male" | "female" | "other" | "no";
+
+type RaceResponse =
+  | "Asian"
+  | "Black or African American"
+  | "Native American or Alaskan Native"
+  | "Native Hawaiian or Other Pacific Islander"
+  | "White"
+  | "Other"
+  | "Prefer not to say";
+
+type EmploymentTypeResponse = "part" | "full";
+
+type CompensationRangeResponse =
```

Comment:
Wondering if these types for describing responses might be safer as [enums](https://www.typescriptlang.org/docs/handbook/enums.html#enums),